solidity/contracts/converter/BancorConverter.sol

```diff
@@ -44,7 +44,7 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
     /**
       * @dev version number
     */
-    uint16 public version = 27;
+    uint16 public version = 28;

```

Comment:
@yudilevi : Actually, yes - first in 0.5.19, and then in this branch.
I believe that we would like this separation, because changes in the converter, even if technically "cosmetic", **have** taken place in 0.5.19.
So in case there are any issues, I think that we would like to be able to version-distinguish between those "cosmetic" changes and the "real" ones.

Reply:
We're going to merge them together with 0.5.19 into master so it's redundant - if we increase it again, there won't ever be 27 in master.

Reply:
@yudilevi : done.